webapp/src/main/java/com/box/l10n/mojito/okapi/PseudoLocalizeStep.java

```diff
@@ -13,10 +13,11 @@
  *
  * @author srizvi
  */
-public abstract class PseudoLocalizeStep extends BasePipelineStep {
+public class PseudoLocalizeStep extends BasePipelineStep {
 
     protected String source;
     protected ITextUnit textUnit;
+    protected PseudoLocalization pseudoloc;
```

Comment:
please rename to  `pseudoLocalization` and you need `@Autowired` to inject the dependency into the class